### PR TITLE
Better example

### PR DESCRIPTION
**Pull request by ghutchis**
## Description
Slight better example with cleaned-up JSON

### CODE REVIEW

File: Examples/simple.json [hunk 1]
```diff
@@ -1,59 +1,63 @@
...
+            "method": "SCF",
+            "basis": "sto-3g",
+            "_myqc-grid": "mygrid123",
+            "_myqc-guess": "harris"
         },
-        'SCF TWO-ELECTRON ENERGY': {
-            'val': 4.124089347186247,
-            'units': 'hartree'
+        "properties": {
+            "SCF N ITERS": 2.0,
```
Comment (cryos) [inline Examples/simple.json]: Do we want to use spaces in keys, or camelCase, or snake_case?

**Comment by wadejong** [comment]
My vote is for camelCase.

File: Examples/simple.json (hunk 1 above)
Comment (langner) [reply Examples/simple.json]: I vote for snake_case.

**Comment by ghutchis** [comment]
I adapted the previous example, which left keys with spaces.

**Pull request merged**